docs/index.md

```diff
@@ -80,7 +80,7 @@ This site and the Open Data Census platform is run by [Open Knowledge Internatio
 
 #### This is great! Can I make a donation?
 
-The Open Knowledge International is a non-profit and we appreciate any and all donations. Support the running of censuses by donating now via Paypal!
+Open Knowledge International is a non-profit and we appreciate any and all donations. Support the running of censuses by donating now via Paypal!
```

Comment:
@pwalsh - this is a great section here, but what are our PayPal details so we can add them here? Or Michelle knows that?